<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_obstacle</class>
 <widget class="QDialog" name="add_obstacle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add_Obstacle</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; font-style:italic; text-decoration: underline;&quot;&gt;Add Obstacles to the system!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="return_c">
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="return_2">
      <property name="text">
       <string>Return To Menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="plan">
      <property name="text">
       <string>Plan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="brique">
      <property name="text">
       <string>Brique</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="position_b">
        <property name="text">
         <string>Get_Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pos_b">
        <property name="text">
         <string>Position! (3 coord)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="lenght_1">
        <property name="text">
         <string>Get_Lenght</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lenght_">
        <property name="text">
         <string>Lenght! (3 coord)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="width_1">
        <property name="text">
         <string>Get_Width</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="width_">
        <property name="text">
         <string>Width! (3 coord)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="height_1">
        <property name="text">
         <string>Get_Height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="height_">
        <property name="text">
         <string>Height! (1 coord)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="position_">
        <property name="text">
         <string>Get_Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="position">
        <property name="text">
         <string>Position!(3 coord)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="normal_">
        <property name="text">
         <string>Get_normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="normal">
        <property name="text">
         <string>Normal!(3 coord)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="go_btn">
      <property name="text">
       <string>GO!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
